sleap/io/dataset.py

```diff
@@ -111,6 +111,7 @@ def rebuild_cache(self):
         self._track_occupancy = dict()
         self._frame_count_cache = dict()
         self._session_by_video: Dict[Video, RecordingSession] = dict()
+        self.linkage_of_videos = {"linked": [], "unlinked": []}
```

Comment:
Let's stick with previous convention and add an underscore:
```suggestion
        self._linkage_of_videos = {"linked": [], "unlinked": []}
```
just so that others know not just to be using/updating this variable wherever (supposedly "private" to the containing class, even though that doesn't exist in python).